plugin/trino-oracle/src/test/java/io/trino/plugin/oracle/AbstractTestOracleTypeMapping.java

```diff
@@ -788,56 +787,73 @@ public void testTimestamp(boolean insertWithTrino, ZoneId sessionZone)
         };
     }
 
-    @Test(dataProvider = "testTimestampWithTimeZoneDataProvider")
-    public void testTimestampWithTimeZone(boolean insertWithTrino)
+    @Test
+    public void testTimestampWithTimeZoneFromTrino()
     {
-        DataType<ZonedDateTime> dataType;
-        DataSetup dataSetup;
-        if (insertWithTrino) {
-            dataType = trinoTimestampWithTimeZoneDataType();
-            dataSetup = trinoCreateAsSelect("timestamp_tz");
-        }
-        else {
-            dataType = oracleTimestamp3TimeZoneDataType();
-            dataSetup = oracleCreateAndInsert("timestamp_tz");
-        }
-
-        DataTypeTest tests = DataTypeTest.create()
-                .addRoundTrip(dataType, epoch.atZone(UTC))
-                .addRoundTrip(dataType, epoch.atZone(kathmandu))
-                .addRoundTrip(dataType, epoch.atZone(fixedOffsetEast))
-                .addRoundTrip(dataType, epoch.atZone(fixedOffsetWest))
-                .addRoundTrip(dataType, beforeEpoch.atZone(UTC))
-                .addRoundTrip(dataType, beforeEpoch.atZone(kathmandu))
-                .addRoundTrip(dataType, beforeEpoch.atZone(fixedOffsetEast))
-                .addRoundTrip(dataType, beforeEpoch.atZone(fixedOffsetWest))
-                .addRoundTrip(dataType, afterEpoch.atZone(UTC))
-                .addRoundTrip(dataType, afterEpoch.atZone(kathmandu))
-                .addRoundTrip(dataType, afterEpoch.atZone(fixedOffsetEast))
-                .addRoundTrip(dataType, afterEpoch.atZone(fixedOffsetWest))
-                .addRoundTrip(dataType, timeDoubledInJvmZone.atZone(UTC))
```

Comment:
here too, i see some value of using fields such as `timeDoubledInJvmZone` (which are verified to have meaning true to their name), even at the cost of using a formatter function instead of literal as the input.